packages/vue-components/src/Submenu.vue

```diff
@@ -2,13 +2,14 @@
   <li
     ref="submenu"
     :class="[addClass, 'dropdown-submenu',
-             { 'dropright': dropright, 'dropleft': dropleft }]"
+             { 'dropend': dropright, 'dropstart': dropleft }]"
```

Comment:
Will need to adjust the CSS of the submenu. Need to remove the extra left padding and change the colour of the text.

![image](https://user-images.githubusercontent.com/47470981/162555696-44c2e540-856e-4cc1-a07a-542aa9d1c8a2.png)